project/profiles/capacity4more/modules/c4m/og/c4m_og_menu/c4m_og_menu.features.menu_custom.inc

```diff
@@ -1,25 +0,0 @@
-<?php
-/**
- * @file
- * c4m_og_menu.features.menu_custom.inc
- */
-
-/**
- * Implements hook_menu_default_menu_custom().
- */
-function c4m_og_menu_menu_default_menu_custom() {
```

Comment:
A different menu implementation is used to allow project managers to customize the menu.
Instead of the previous `/admin/structure/menu` path, you'll find the menu settings at `/admin/config/group/og_menu` with the default menu items at `/admin/config/group/og_menu/default-links`.